style.css

```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #191765;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2.5em;
+}
+
+.myLogo {
+  color: white;
+  text-decoration: none;
+  font-size: 18px;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+}
+
+#whiteEllipse {
+  display: block;
+  float: left;
+}
+
+#myName {
+  color: #7d73fc;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 48px;
+  text-align: center;
+}
+
+#myTitle {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+  color: #edecff;
+  text-align: center;
+}
+
+.shortBio {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  color: #c8c8d4;
+  text-align: center;
+  margin-top: 50px;
+  margin-bottom: 30px;
+}
+
+#myTitle:hover,
+#myName:hover,
+.shortBio:hover,
+.myLogo:hover {
+  text-decoration: underline;
+  text-decoration-color: #0d24f0;
+}
+
+.headline {
+  background-image: url(/assets/ellipseHeader.svg);
```

Comment:
- [ ] There is an issue with your background because it's not showing. I think it is because of the `/`, you could try removing it or even better adding a `.` before so it look like this: `background-image: url(./assets/ellipseHeader.svg);`

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/143484242-c043208b-68c1-47e2-b47e-6481b2123098.png)

  With the `.` before:
  ![image](https://user-images.githubusercontent.com/81522435/143484360-eee6ced8-db31-49ea-8e29-b3a82d8e3590.png)
  ![image](https://user-images.githubusercontent.com/81522435/143484384-a9960b1d-6a8b-4ec2-b1eb-477fea712e82.png)